web/assets/css/transmission-app.scss

```diff
@@ -933,14 +933,17 @@ a {
   flex-direction: column;
   position: absolute;
   right: 0;
-  top: var(--popup-top);
```

Comment:
I'm seeing other media queries declare each of the same properties instead of having "viewport neutral" properties above them. I see CSS is in need of a good cleanup so I'll put them back and then we'll discuss what to do about duplicate properties in another PR.